gobblin-api/src/main/java/gobblin/writer/DataWriterBuilder.java

```diff
@@ -25,11 +27,17 @@
  */
 public abstract class DataWriterBuilder<S, D> {
 
+  @Getter
```

Comment:
The constructor signature of some writers are changed from `(State properties, String fileName, Schema schema, int numBranches, int branchId)` to `(FsDataWriterBuilder<?, D> builder, State properties)`. They now need to obtain `fileName, schema, numBranches, branchId` from the builder. It is currently fine since those writers and builders are in the same package, but it is possible that in the future a writer is added in a different package.